R/FilteredData.R

```diff
@@ -93,10 +94,10 @@ FilteredData <- R6::R6Class( # nolint
   public = list(
     #' @description
     #' Initialize a `FilteredData` object.
-    #' @param data_objects (`list`)
-    #'   Named list of data objects.
-    #'   Names of the list will serve as `dataname`.
-    #' @param join_keys (`join_keys` or NULL) see [`teal.data::join_keys()`].
+    #' @param data_objects (`named list`)
+    #'   of data objects.
+    #'   Names of the list will be used as `dataname` for respective datasets.
+    #' @param join_keys (`join_keys`) optional joining keys, see [`teal.data::join_keys()`].
```

Comment:
Recycling a comment about using ``(`named list`)`` as part of the sentence. Is this deliberate?

Reply:
Fair point.